docker-compose.yml

```diff
@@ -6,9 +6,9 @@ services:
     container_name: task_database
     restart: always
     environment:
-      MYSQL_USER: e-edu
-      MYSQL_PASSWORD: e-edu
-      MYSQL_DATABASE: e-edu
+      MYSQL_USER: gewia
```

Comment:
Does the server admin know about the changes?

Reply:
Probably not I'll change it